docs/changes/2629.feature.rst

```diff
@@ -0,0 +1,15 @@
+This commit adds two new hillas-parameter variables:
+1. psi_uncertainty (uncertainty on the psi angle of the image)
+2. transverse_cog_uncertainty (uncertainty on the center of gravity along the transverse axis of the image)
+
+The method that estimates these uncertainties has been presented at these ASWG calls:
+https://indico.cta-observatory.org/event/5847/contributions/47481/attachments/26575/39053/ctapipe_rethink_hillas_20240718.pdf
+https://indico.cta-observatory.org/event/5900/contributions/47899/attachments/26789/39379/Psi_uncertainty_short_summary_20241022.pdf
+
+The affected files are:
+image/hillas.py
+containers.py
+
+Caution:
```

Comment:
This does not belong in a changelog but in the actual documentation of the method. Also: how is this information to be used?